packages/@okta/vuepress-site/docs/guides/pwd-optional-best-practices/main/index.md

```diff
@@ -12,8 +12,10 @@ The phone authenticator is a popular and recommended alternative authenticator b
 
 ## Protect your admin accounts
 
-Before you configure the password-optional experience, ensure that your admins continue to have passwords available. This guarantees that users who don't have a password provisioned are not inadvertently blocked from signing in. To do this:
+Before you configure the password-optional experience, ensure that passwords continue to be required for your admins. Should a passwordless account be granted admin privileges at some point, they will then be required to create a password giving them access to all your Org's apps. To do this:
 
 * Create a separate group for admins and add your admin users to that group.
 * Create separate authenticator enrollment, global session, and authentication policies for this group.
```

Comment:
The line that says "This will also prevent new admin accounts being created without a password." is misleading because you can create a profile enrollment and global policy that allows for no passwords and bypass the Admin Group password requirements.

Reply:
True. Have deleted the line.